LEAF_Nexus/templates/permission_iframe.tpl

```diff
@@ -1,26 +1,26 @@
 <!--{foreach from=$privileges item=privilege}-->
     <!--{if $privilege.read == 0}-->
-<div style="padding: 4px; color: red">
+<div tabindex="0" style="padding: 4px; color: red">
```

Comment:
Please add role="button" so the screen readers read them as buttons

Reply:
Done